spec/std/zip/zip_serializer_spec.cr

```diff
@@ -0,0 +1,533 @@
+require "../spec_helper"
+require "zip"
+
+class ByteReader
+  def initialize(io : IO)
+    @io = io
+    @io.rewind
+  end
+
+  def read_1b
+    slice = read_n(1)
+    slice[0]
+  end
+
+  def read_2b
+    @io.read_bytes(UInt16, format = IO::ByteFormat::LittleEndian)
```

Comment:
`format =` -> `format:` (former way is declares local variable named `format` and passes it as an positional argument, while the latter passes `format` as named argument, which I reckon is what you wanted)

ditto all below